### PR TITLE
copyright changes

### PR DESCRIPTION
**Pull request by YiJie-Zhu**

### CODE REVIEW

File: lib/presentation/engage/bible_series/widgets/scripture_body.dart [hunk 1]
```diff
@@ -81,7 +81,8 @@ Widget book(Scripture script, List<int> sortedVerses) {
 Widget copyright(ScriptureBody script) {
   String attribution = script.properties.attribution;
   String version = script.properties.bibleVersion;
-  String text = "$version@\n$attribution";
+  String text =
+      "Scripture quotations taken from $version\n$attribution\nUsed by permission. All rights reserved worldwide";
```
Comment (JoshuaRamsamooj) [inline lib/presentation/engage/bible_series/widgets/scripture_body.dart]: We should be able to make this change in Firebase right?

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: The version and the attribution can be changed. This is just the format that NIV recommends on their website.

Comment (JoshuaRamsamooj) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: I'm wondering if we can change bible_version in Firebase from `The Holy Bible, New International Version® NIV®` to `Scripture quotations taken from The Holy Bible, New International Version® NIV®` and something similar for attribution.

Comment (JoshuaRamsamooj) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: Just so that we have full control of this in Firebase

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: I see what you mean. That makes sense, I'll go make those changes.

Comment (JoshuaRamsamooj) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: Do we still need this PR then?

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: just for the "Used by permission. All rights reserved worldwide"

**Comment by JoshuaRamsamooj** [comment]
Changes to be made directly in Firestore

**Pull request closed without merging**